aviary/core/aviary_group.py

```diff
@@ -1245,10 +1245,12 @@ def add_design_variables(self, verbosity=None):
             # fixed vehicle (design GTOW) but variable actual GTOW for off-design
             # mission range
             elif self.problem_type is ProblemType.ALTERNATE:
+                # get the design gross mass and set as the upper bound for the gross mass design variable
+                MTOW = self.aviary_inputs.get_val(Mission.Design.GROSS_MASS, 'lbm')
                 self.add_design_var(
                     Mission.Summary.GROSS_MASS,
                     lower=10.0,
-                    upper=900e3,
+                    upper=MTOW,
```

Comment:
When you run the design mission, the `aviary_inputs` contains the initial guess for the design gross mass. Does that value get updated in aviary_inputs somewhere in this process prior to this point?